Formula/subversion.rb

```diff
@@ -223,8 +225,12 @@ def install
           "-DPERL_DARWIN -fno-strict-aliasing -I#{HOMEBREW_PREFIX}/include -I#{perl_core}"
       end
     end
-    system "make", "swig-pl"
-    system "make", "install-swig-pl"
+    system "make", "swig-pl-lib"
+    system "make", "install-swig-pl-lib"
+    cd "subversion/bindings/swig/perl/native" do
+      system perl, "Makefile.PL", "PREFIX=#{prefix}", "INSTALLSITEMAN3DIR=#{man3}"
+      system "make", "install"
+    end
```

Comment:
Alternatively could have
- manually moved files (`man3.install ...`) or
- modified the Makefile
    ```rb
    inreplace "Makefile", "$(PERL) Makefile.PL PREFIX=$(prefix)", "\\0 INSTALLSITEMAN3DIR=#{man3}"
    ```

The current approach is officially documented method for adding options during Perl build https://github.com/apache/subversion/blob/1.14.2/subversion/bindings/swig/INSTALL#L217-L236

`PREFIX` is from the Makefile https://github.com/apache/subversion/blob/1.14.2/Makefile.in#L883-L884 while `INSTALLSITEMAN3DIR` is to fix the manpage installation.

Reply:
The documented approach looks pretty clean so it seems fine to me.